src/Components/Publishing/Fixtures/Articles.ts

```diff
@@ -805,6 +805,7 @@ export const NewsArticle: ArticleData = {
     id: "50f4367051e7956b6e00045d",
     name: "Artsy Editorial",
   },
+  date: "2018-07-19T17:19:55.909Z",
   authors: [
```

Comment:
Our api uses `published_at`, and `scheduled_publish_at` as date fields -- not sure if this is replicating return values from graphql, but might make sense to use a different field title for fixtures so it is more compatible with Positron.